src/common/intl/index.js

```diff
@@ -88,6 +90,10 @@ export class FormattedDuration extends Component {
       duration,
       lang
     } = this.props
-    return <span>{moment.duration(duration).locale(lang).humanize()}</span>
+    const momentDuration = moment.duration(duration)
+
+    return <Tooltip content={getMessage('parseDuration', parseDuration(momentDuration.asSeconds()))}>
+      <span>{momentDuration.locale(lang).humanize()}</span>
```

Comment:
Maybe we should memoize `parseDuration(momentDuration.asSeconds())` and `momentDuration.locale(lang).humanize()`?